GVRf/tools/Framework/gradle.properties

```diff
@@ -16,7 +16,7 @@
 org.gradle.jvmargs=-Xmx2048M
 #ANDROID_NDK_HOME=
 # Un-comment and add the path to ovr directory
-#OVR_MOBILE_SDK=
+OVR_MOBILE_SDK=d:/projects/android/GearVRf/ovr_sdk_mobile
```

Comment:
You could define properties in the global gradle.properties; under Linux, modify ~/.gradle/gradle.properties.